pkg/ccl/backupccl/generative_split_and_scatter_processor.go

```diff
@@ -223,14 +239,15 @@ func runGenerativeSplitAndScatter(
 	ctx context.Context,
 	flowCtx *execinfra.FlowCtx,
 	spec *execinfrapb.GenerativeSplitAndScatterSpec,
-	scatterer splitAndScatterer,
+	scatterers []splitAndScatterer,
+	secondSplitScatterers []splitAndScatterer,
 	doneScatterCh chan<- entryNode,
 ) error {
 	log.Infof(ctx, "Running generative split and scatter with %d total spans, %d chunk size, %d nodes",
 		spec.NumEntries, spec.ChunkSize, spec.NumNodes)
 	g := ctxgroup.WithContext(ctx)
 
-	splitWorkers := int(spec.NumNodes)
+	splitWorkers := len(scatterers)
```

Comment:
nit: maybe we call this splitAndScatterWorkers since they do both?

Reply:
done